src/coreclr/vm/codeman.cpp

```diff
@@ -2517,7 +2517,11 @@ HeapList* LoaderCodeHeap::CreateCodeHeap(CodeHeapRequestInfo *pInfo, LoaderHeap
     DWORD dwSizeAcquiredFromInitialBlock = 0;
     bool fAllocatedFromEmergencyJumpStubReserve = false;
 
-    pBaseAddr = (BYTE *)pInfo->m_pAllocator->GetCodeHeapInitialBlock(loAddr, hiAddr, (DWORD)initialRequestSize, &dwSizeAcquiredFromInitialBlock);
+    size_t allocationSize = pCodeHeap->m_LoaderHeap.AllocMem_TotalSize(initialRequestSize);
+#if defined(TARGET_AMD64) || defined(TARGET_ARM64) || defined(TARGET_LOONGARCH64)
```

Comment:
Can we use `#ifdef TARGET_64BIT` instead (as done below on line 2600)?

Reply:
Sure, I can do that. I've just copied the same ifdef that we have below where we allocate the personality routine. But there are many more places in the code where we have this same compound `#if`. So maybe it would be worth a separate cleanup instead.

Reply:
Sure. Some of the usage of `HOST_64BIT` in VM can also be reviewed as part of that work, because it looks like the correct macro for them (in most places) is `TARGET_64BIT` (but it will require a in-depth analysis: https://github.com/dotnet/runtime/pull/2256#issuecomment-579543182).

Reply:
I've created issue #71220 for that.